Documents/FunctionalSpecifications/functionalSpecifications.md

```diff
@@ -49,6 +49,7 @@
         - [The Game](#the-game)
             - [The environment](#the-environment)
             - [The player](#the-player)
+        - [The Console](#the-console)
     - [4. Personas definition](#personas-definition)
```

Comment:
not linked (as well as cars)

Reply:
cars yes it doesn't work but this one you showed works

Reply:
fixed, i let you check if it's good

Reply:
cars is alright, though personas definition still does not work

Reply:
persona not working